### PR TITLE
MG1933073

### PR DESCRIPTION
**Pull request by ronleeon**
Class,202 same
Arrays,246 same
String,158 same

### CODE REVIEW

**Comment by zhangruimr** [comment]
MP1933027

**Comment by zhangruimr** [comment]
MP1933027 张锐 评估已通过

File: MG1933073/java/Class_getSimpleName.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * @description:Test java api java.lang.Class.getSimpleName()
+ * @Map:Objective-C Runtime$func class_getName(_ cls: AnyClass?) -> UnsafePointer<Int8>
+ */
+    public class Class_getSimpleName{
+        /**
+         * input:1
+         * class0 class0="Hello,World"
+         * output:1
+         * ret0 ret0="String"
+         */
```
Comment (zjzh) [inline MG1933073/java/Class_getSimpleName.java]: ret0数据类型错误

File: MG1933073/swift/Array_prefix_Int__Array_suffix_Int__NSMutableArray_sort_NSSortOptions_ComparisonResult.swift [hunk 2]
```diff
@@ -0,0 +1,141 @@
+/**类名_方法名(_形参类型)*
+ * @author:MG1933073 熊俊
+ * @className:Array_prefix_Int__Array_suffix_Int__NSMutableArray_sort_NSSortOptions_ComparisonResult
+ * @apiSignature:Array$func prefix(upTo end: Int) -> ArraySlice<Element>;Array$func suffix(from start: Int) -> ArraySlice<Element>;NSMutateArray$func sort(options opts: NSSortOptions=[],usingComparator cmptr:(Any,Any)->ComparisonResult)
+ * @description:Test swift api Array$prefix(upTo end: Int);Array$suffix(from start: Int);NSMutableArray$sort(options opts: NSSortOptions=[],usingComparator cmptr:(Any,Any)->ComparisonResult)
+ * @Map:java.util.Arrays$public static void sort(Object[] a,int fromIndex,int toIndex)
+ */
+ import Foundation
+ class Array_prefix_Int__Array_suffix_Int__NSMutableArray_sort_NSSortOptions_ComparisonResult{
```
Comment (idiomaticrefactoring) [inline MG1933073/swift/Array_prefix_Int__Array_suffix_Int__NSMutableArray_sort_NSSortOptions_ComparisonResult.swift]: 类名错误

File: MG1933073/swift/ObjectiveCRuntime_class_getName_AnyClass.swift [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * @Map:java.lang.Class$public String getSimpleName()
+ */
+ import Foundation
+ class ObjectiveCRuntime_classgetName_AnyClass{
+     /**
+      *input:1
+      *cls cls="NSString"
+      *output:1
+      *ret0 ret0="NSString"
+      */
```
Comment (idiomaticrefactoring) [inline MG1933073/swift/ObjectiveCRuntime_class_getName_AnyClass.swift]: ret0类型错误

**Review by idiomaticrefactoring — CHANGES_REQUESTED** [review]
perfect-1

**Pull request closed without merging**